### PR TITLE
issue_template: add instructions to get pkg version

### PR DESCRIPTION
**Pull request by anjandev**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 ### System
 
 * xuname:  
-  *output of ``xuname`` (part of xtools)*
+  - *output of ``xuname`` (part of xtools)*
 * package:  
-  *affected package(s) including the version*
+  - *affected package(s) including the version.*
+  - *output of ``xbps-query -Rs <pkgname> | grep pkgver``*
```
Comment (not-chicken) [inline .github/issue_template.md:8]: Shouldn't this be without `-s`?
Also I'd prefer this:
```
xbps-query -R <pkgname> --property=pkgver
```

Comment (not-chicken) [reply .github/issue_template.md:8]: i.e. `xbps-query -R <pkgname> -p pkgver` .

**Comment by anjandev** [comment]
Good catch. Fixed. Thank you

**Comment by not-chicken** [comment]
One more issue, this command will return the latest version available in repositories which may not be the one that's installed on user's system.

Dropping -R solves that but will make it return nothing in case user's issue is about not being able to install the package ( due to conflicts, etc).

-------- Original Message --------

File: .github/issue_template.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 ### System
 
 * xuname:  
-  *output of ``xuname`` (part of xtools)*
+  - *output of ``xuname`` (part of xtools)*
 * package:  
-  *affected package(s) including the version*
+  - *affected package(s) including the version.*
+  - *output of ``xbps-query <pkgname> -p pkgver``*
```
Comment (Chocimier) [inline .github/issue_template.md:8]: This is same, so why two items of list?

**Comment by anjandev** [comment]
> This is same, so why two items of list?

I remove the old item from the list.

-- 
w:] www.momi.ca
pgp:] https://momi.ca/publickey.txt

**Comment by Chocimier** [comment]
Reformatted and merged.

**Pull request merged**